<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>216</width>
    <height>89</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/res/images/favicon/favicon_16x16.png</normaloff>:/icons/res/images/favicon/favicon_16x16.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>216</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionRemove"/>
   </widget>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionRollback"/>
    <addaction name="actionCommit"/>
    <addaction name="separator"/>
    <addaction name="actionExport"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuDatabase">
    <property name="title">
     <string>&amp;Database</string>
    </property>
    <addaction name="actionPrevious"/>
    <addaction name="actionNext"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menuDatabase"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRollback"/>
   <addaction name="actionCommit"/>
   <addaction name="separator"/>
   <addaction name="actionAdd"/>
   <addaction name="actionRemove"/>
   <addaction name="separator"/>
   <addaction name="actionPrevious"/>
   <addaction name="actionNext"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/menuEdit/res/images/listAdd.png</normaloff>:/menuEdit/res/images/listAdd.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Add</string>
   </property>
   <property name="toolTip">
    <string>Add</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionRemove">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/menuEdit/res/images/listRemove.png</normaloff>:/menuEdit/res/images/listRemove.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Remove</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionRollback">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/menuFile/res/images/rollBack.png</normaloff>:/menuFile/res/images/rollBack.png</iconset>
   </property>
   <property name="text">
    <string>Rollback</string>
   </property>
  </action>
  <action name="actionCommit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/menuFile/res/images/commit.png</normaloff>:/menuFile/res/images/commit.png</iconset>
   </property>
   <property name="text">
    <string>Commit</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/menuDatabase/res/images/next.png</normaloff>:/menuDatabase/res/images/next.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Next</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/menuDatabase/res/images/previous.png</normaloff>:/menuDatabase/res/images/previous.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Previous</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
